app.conf.template

```diff
@@ -13,6 +13,11 @@ JWT_SECRET_KEY = 'jwt-secret-string'
 RABBIT_MQ_URL= 'amqp://user:leftfoot1@localhost:30672/%2F'
 TRANSFORMER_RABBIT_MQ_URL= 'amqp://user:leftfoot1@host.docker.internal:30672/%2F?heartbeat=9000'
 
+# Keep retrying every five seconds for two minutes before giving up
```

Comment:
Should this match the 12 retries at 10-second intervals from the values.yaml?

Reply:
Yes, once I started using the helm chart I decided every five seconds was a little too aggressive.